cluster_provisioning/ansible/playbooks/build-sync-gateway-source.yml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
We should add support to directly install sync-gateway from a given url, example from latestbuild like we have support for couchbase server installer.